maintainers/maintainer-list.nix

```
@@ -3765,6 +3765,12 @@
     githubId = 2396926;
     name = "Justin Woo";
   };
+  jwatt = {
+    email = "jwatt@broken.watch";
+    github = "jjwatt";
+    githubId = "2397327";
```

```suggestion
    githubId = 2397327;
```

Just to fall in line with the rest of the file ;)